### ISSUE TITLE
In the grid, ctrl + a does not select all.

### ISSUE THREAD

**Issue by engca** [nhn/tui.grid#1446]
**Describe the bug**
When adding data with appendRows in vue mounted(), ctrl + a does not select all.

**To Reproduce**
1. Go to https://codesandbox.io/s/boring-morning-6g3w8?file=/src/App.vue:1144-1154
2. Click on 'grid1' , ctrl + a
3. All selected
4. Click on 'grid2' , ctrl + a
5. Not all selected

**Expected behavior**
You can select all with ctrl + a

**Screenshots**
Same as #1441

**Desktop (please complete the following information):**
  - OS: Windows 10
  - Browser chrome
  - Version 14.17.2

**Additional context**
This is what I previously asked a question on #1441.
The same symptom occurs in version 14.18.1 as well.

**Comment by engca** [comment]
Hello.
I don't have an answer yet, but I'll leave it as a reply to what I find additionally.
If there is already data in the grid, the above symptom does not appear even if appendRows is performed, and the function to select all with ctrl + a works normally.
https://codesandbox.io/s/frosty-neumann-c2suc?file=/src/App.vue
You can check it out here.
thank you.

**Comment by js87zz** [comment]
@engca 
This is an issue caused by the remaining reference to the shared data.
The each data should be independent as below. 
```js
function createData() {
  return [
      {
        name: "Beautiful Lies",
        artist: "Birdy",
      },
      {
        name: "X",
        artist: "Ed Sheeran",
      },
      {
        name: "Beautiful Lies",
        artist: "Birdy",
      },
      {
        name: "X",
        artist: "Ed Sheeran",
      },
    ];
}
// ...
mounted() {
  const data = createData();
  data.forEach((value) => {
    this.$refs.grid1.invoke("appendRow", value);
  });
  this.$refs.grid2.invoke("appendRows", createData());
}

```

**Comment by engca** [comment]
Hello.
I applied and tested as you replied, but the same symptom occurs.
Am I not applying it properly?
If so, I would appreciate it if you could tell me the relevant keywords.
https://codesandbox.io/s/hopeful-sea-gv0gv?file=/src/App.vue
The above symptoms can be checked here.
thank you.

**Comment by js87zz** [comment]
@engca 
Oh..I checked the issue. 
If the issue is fixed, I will comment here.

**Comment by js87zz** [comment]
@engca 
The issue has been fixed in `v4.19.1`.
